src/Modal/Modal.js

```diff
@@ -3,14 +3,27 @@ import './Modal.scss';
 
 
 function Modal({foundMovie, closeModal}) {
-
+  console.log(foundMovie)
   return (
     (!foundMovie) ? <p>No movie clicked yet</p> : 
     <section className="modal">
       <img src={foundMovie.backdrop_path} alt="movie backdrop"/>
-      <h2>{foundMovie.title}</h2>
-      <p>Release Date: {foundMovie.release_date}</p>
-      <p>Average Rating: {foundMovie.average_rating}</p>
+      <div className="m-data">
+        <p className="m-tagline">{foundMovie.tagline}</p>
+        <h2 className="m-title">{foundMovie.title}</h2>
+        <p className="m-date">Release Date: {foundMovie.release_date}</p>
+        <p className="m-rating">Average Rating: {foundMovie.average_rating.toFixed(1)}</p>
+        <p className="m-runtime">Runtime: {foundMovie.runtime} minutes</p>
+        <p className="m-overview">{foundMovie.overview}</p>
+        <div className="m-genre">
+          <p>Filed under:</p>
+            <ul>
+              {foundMovie.genres.map(genre => <li>{genre}</li>)}
+            </ul>
+        </div>
+        <p className="m-budget">Budget: ${foundMovie.budget.toFixed(2)}</p>
+        <p className="m-revenue">Revenue: ${foundMovie.revenue.toLocaleString()}</p>
```

Comment:
are the chained methods considered "functions" inside our returns?

Reply:
Good question, I could see it either way for these - since they're built-in methods and they're only for display, I would push for "this is ok".